.nox/design/sample.solution.nox.yaml

```diff
@@ -10,10 +10,10 @@ description: Sample Nox solution yaml configuration
 
 variables:
    DATABASE_PROVIDER: slqServer   
-   DATABASE_SERVER: SamplewebAppDb
+   DATABASE_SERVER: 127.0.0.1
```

Comment:
you need to revert this line, we need to double check with @jan-schutte 
its related to the new docker project